src/Microsoft.Health.Fhir.Shared.Api/Modules/SearchParametersModule.cs

```diff
@@ -0,0 +1,33 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.Extensions.Hosting;
+using Microsoft.Health.Extensions.DependencyInjection;
+using Microsoft.Health.Fhir.Core.Features.Search.Registry;
+using Microsoft.Health.Fhir.Core.Registration;
+
+namespace Microsoft.Health.Fhir.Api.Modules
+{
+    /// <summary>
+    /// Registration of search components.
+    /// </summary>
+    public static class SearchParametersModule
+    {
+        public static IFhirServerBuilder AddSearchParametersModules(this IFhirServerBuilder fhirServerBuilder)
```

Comment:
>this IFhirServerBuilder [](start = 68, length = 23)

nit: I found only one extension on top of `IFhirServerBuilder` and it's in `FhirServerServiceCollectionExtensions`, not sure is it worth to create new class. Maybe we could add that code next to `AddBackgroundWorkers`